### DISCUSSION TITLE
Frigate Versions?

### DISCUSSION THREAD

**Discussion by ksignorini** [blakeblackshear/frigate#2382] [Ask A Question]
I'm rebuilding some things and just realized there are different versions of Frigate that I can install. Which of these is the right one? What's the diff between regular and Full Access? 

![image](https://user-images.githubusercontent.com/6867660/144736066-77c5116c-b48c-44ac-acac-0214616a1e7f.png)

**Comment by blakeblackshear** [comment]
Full Access allows you to turn off protection mode if needed. It has a lower security rating.

**Reply by egamper** [reply]
Hi on my HA with Frigate NVR (Full Acccess) i can access Frigate via http://homeassistant.local:5000 without authN. Is this related to the Full Access version of the Add on? THX

**Reply by NickM-27** [reply]
> Hi on my HA with Frigate NVR (Full Acccess) i can access Frigate via http://homeassistant.local:5000 without authN. Is this related to the Full Access version of the Add on? THX

The full access has to do with the addons (frigate's) access to HA OS resources. Meaning it can access more of the system hardware and services. 

It's not referring to users'access to frigate.
